src/resolve/resolved-dns-netservice-gperf.gperf

```diff
@@ -0,0 +1,23 @@
+%{
+#include <stddef.h>
+#include "conf-parser.h"
+#include "resolved-conf.h"
+#include "resolved-dns-netservice.h"
+%}
+struct ConfigPerfItem;
+%null_strings
+%language=ANSI-C
+%define slot-name section_and_lvalue
+%define hash-function-name resolved_netservice_gperf_hash
+%define lookup-function-name resolved_netservice_gperf_lookup
+%readonly-tables
+%omit-struct-type
+%struct-type
+%includes
+%%
+Service.Name,     config_parse_string,              0, offsetof(DnsNetservice, instance_name)
+Service.Type,     config_parse_string,              0, offsetof(DnsNetservice, type)
```

Comment:
i figure these should have explicit parsers, that validate that the type is actually really valid, given that there's a strict syntax regime defined for these.